demos/common/monitors/presenter.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <map>
+#include <ostream>
+#include <set>
+
+#include <opencv2/imgproc.hpp>
+
+#include "cpu_monitor.h"
+#include "memory_monitor.h"
+
+enum class MonitorType{CpuAverage, DistributionCpu, Memory};
+
+void meansToOstream(const std::map<MonitorType, std::vector<double>>, std::ostream& stream);
+
+class Presenter {
+public:
+    explicit Presenter(std::set<MonitorType> enabledMonitors = {},
+        int yPos = 20,
+        cv::Size graphSize = {150, 60},
+        std::size_t historySize = 20);
+    explicit Presenter(const std::string& keys,
+        int yPos = 20,
+        cv::Size graphSize = {150, 60},
+        std::size_t historySize = 20);
+    void addRemoveMonitor(MonitorType monitor);
+    void addRemoveMonitor(int key); // handles c, d, m, h keys
```

Comment:
I think this is the only way presenter is going to handle a key, which allows to clarify in function name how the key is handled.

Reply:
I think this is the only way presenter is going to handle a key, which allows to clarify in function name how the key is handled.

Reply:
> I think this is the only way presenter is going to handle a key

It is not the only way already, since the presenter handles the H key by toggling _multiple_ monitors.

Reply:
renamed